charts/kcp/values.yaml

```diff
@@ -21,6 +41,8 @@ virtualWorkspaces:
 audit:
   enabled: true
   volumeSize: 1Gi
+  volumeClassName: ""
+  volumeMode: ""
```

Comment:
I can't think of a scenario where someone would need to specify the `volumeMode`.  I'm sure I had it in the template because it got added by a controller after I created the initial PVC and I saved that version to the template.